backend/src/main/java/org/cryptomator/hub/api/AuditLogResource.java

```diff
@@ -127,6 +123,10 @@ record AuditEventVaultAccessGrantDto(long id, Instant timestamp, String type, @J
 										 @JsonProperty("authorityId") String authorityId) implements AuditEventDto {
 	}
 
+	record AuditEventVaultKeyRetrieveDto(long id, Instant timestamp, String type, @JsonProperty("retrievedBy") String retrievedBy, @JsonProperty("vaultId") UUID vaultId,
```

Comment:
Re naming, I would suggest either `RetrieveKey` or `KeyRetrieval`.

Reply:
> But if this leads to confusion ...

Indeed, I read it like `AuditEvent-Vault-KeyRetrieve` 😅

For me, it is not `AuditEvent-Noun-Verb` but rather `AuditEvent-Domain-Event` with this belonging into the "Vault" domain/namespace.

Hence, the following would be my preferred naming scheme.

> * `AuditEvent-Vault-Create`
> * `AuditEvent-Vault-GrantAccess`
> * `AuditEvent-Vault-RetrieveKey`
> * `AuditEvent-Vault-AddMember`

That said, as already noted in https://github.com/cryptomator/hub/pull/213#pullrequestreview-1536702327, when I asked you to prefix the _table names_, I really didn't want to force the naming scheme upon all entities, DTOs and .ts types, since those would be namespaced by means of the used programming language anyway. 😉